docs/components/google/google-account-picker.md

```diff
@@ -0,0 +1,38 @@
+# Google Account Picker
+
+## Overview
+Google Account Picker is a non-visible component which can be used to pick a Google (Gmail) account registered in the user's device. This component can be used to create a login/register like system in the app.
+<br><br>
+![Google Account Picker](/docs/assets/images/google/preview.gif)
+
+
+## Properties
+This component doesn't have any designer property.
+
+
+
+## Methods
+
+
+### Pick
+
+![Picked](/docs/assets/images/google/m_pick.jpg)

```

Comment:
All images are wrong placed. The path should be `/assets/images/google/google-account-picker/m_pick.jpg`

```suggestion
![Picked](/assets/images/google/google-account-picker/m_pick.jpg)
```

Reply:
But if I don't add "/docs" to it, it is not showing the images.

Reply:
BTW, I have changed the path to `/assets/images/google/google-account-picker` as you stated above.